lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/eventconfig/LambdaUpdateEventConfigurationTask.java

```diff
@@ -176,7 +176,6 @@ private LambdaUpdateEventConfigurationTaskOutput updateEventConfiguration(Lambda
 
         final List<String> existingEvents = getExistingEvents(lf, targetArn);
         taskInput.getTriggerArns().stream()
-               .filter(curr -> { return !existingEvents.contains(curr); })
```

Comment:
this filter blocks updating existing event source mappings; I'm not sure why this was in here?